src/dynamic/hello-bottom/#index.html#

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```

Comment:
This file probably shouldn't be in the commit - with the leading/trailing `#` it looks like it's some kind of editor backup?